doc/jsk_perception/nodes/colorize_float_image.md

```diff
@@ -6,8 +6,12 @@ Colorize float image with heatmap gradation.
 ## Subscribing Topic
 * `~input` (`sensor_msgs/Image`)
 
-  Float(32FC1) image.
+  Float(32FC[1-4]) image.
 ## Publishing Topic
 * `~output` (`sensor_msgs/Image`)
 
   RGB8 image with heatmap gradation.
+## Parameters
+* `~use_channel` (int, default: -1)
+
+  Choose channel to colorize. If value is default, first channel is selected.
```

Comment:
Nice feature of colorize_float_image to select the channel to colorize.
FYI, https://github.com/ros-perception/vision_opencv/pull/150 will expand the feature of `CV_32FC[1-4]` to `CV_32FC[1-N]`.